index_layer[1].html

```diff
@@ -20,6 +20,10 @@
     text-decoration:bold;
     line-height:36px;
 }
+
+.democratie {
+  color: rgb(0, 51, 255);
+}

```

Comment:
And here is the CSS rule to color the label.
I retrieved the color of the layer by inspecting the element with Chrome.

![image](https://cloud.githubusercontent.com/assets/932583/13929511/c7678188-ef9a-11e5-9925-e1ba3029d5fa.png)